src/core/dbus.c

```diff
@@ -753,13 +753,15 @@ int manager_sync_bus_names(Manager *m, sd_bus *bus) {
                         /* If it is, determine its current owner */
                         r = sd_bus_get_name_creds(bus, name, SD_BUS_CREDS_UNIQUE_NAME, &creds);
                         if (r < 0) {
-                                log_error_errno(r, "Failed to get bus name owner %s: %m", name);
+                                if (r != -ENXIO)
+                                        log_error_errno(r, "Failed to get bus name owner %s: %m", name);
```

Comment:
I am pretty sure we should still log in this case, albeit at a lower level. Hence, please change this to:

```c
log_full_errno(r == -ENXIO ? LOG_DEBUG : LOG_WARNING, …
```

or so